index.js

```diff
@@ -188,12 +187,14 @@ function watch({ seq }) {
           // we want to start over and get all info again
           // we do this by exiting and letting Heroku start over
           if (now - lastBootstrapped > c.timeToRedoBootstrap) {
-            stateManager.set({
-              seq: 0,
-              bootstrapDone: false,
-            }).then(() => {
-              process.exit(0);
-            });
+            stateManager
+              .set({
```

Comment:
This is because i did that change in the last PR (#40) with the github web editor, and thus without commit hook.